sdcm/sct_runner.py

```diff
@@ -221,6 +221,7 @@ def install_prereqs(self, public_ip: str, connect_timeout: Optional[int] = None)
              # Disable apt-key warnings and set non-interactive frontend.
             export APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1
             export DEBIAN_FRONTEND=noninteractive
+            export PIP_BREAK_SYSTEM_PACKAGES=true
```

Comment:
I wonder why it started to be needed

Reply:
it's python 3.12, which is the default in ubuntu 2404